src/CodingBat/Warmup_1/frontBackTask_11.java

```diff
@@ -0,0 +1,12 @@
+package CodingBat.Warmup_1;
+
+public class frontBackTask_11 {
+    public static void main(String[] args) {
+
+    }
+
+   // public static String frontBack(String str) {
```

Comment:
remove comments

Reply:
what are you doing?

Reply:
j47rgrwiufyg

Reply:
rtwgwt4g

Reply:
4w5h4w5h

Reply:
sleep

Reply:
sdjf;klsdjl;fkjsd;fjsd;f

Reply:
![png-clipart-spongebob-squarepants-creature-from-the-krusty-krab-wii-patrick-star-spongebob-s-atlantis-squarepantis-mr-krabs-others-miscellaneous-game](https://github.com/Dilnaza07/My_first_project/assets/160740980/b9fb290d-2b9c-490a-a2b7-42abf10a49ec)

Reply:
Gumburgers

Reply:
who is he?

Reply:
![Uploading Avatar for GitHub.jpg…]()

Reply:
don't trouble me

Reply:
![png-clipart-spongebob-squarepants-creature-from-the-krusty-krab-wii-patrick-star-spongebob-s-atlantis-squarepantis-mr-krabs-others-miscellaneous-game](https://github.com/Dilnaza07/My_first_project/assets/160740980/cca4fc38-8a0f-4e92-92c6-33f1e71330a8)

Reply:
![6](https://github.com/Dilnaza07/My_first_project/assets/160740980/f1a6b52d-a547-42f2-8b72-7ccf258cfa60)
SPONGEBOB ATTACK MY ENEMIES!!!!
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

Reply:
Food it's very delicious Spongbob